src/components/dialogs/line-split-with-voltage-level-dialog.js

```diff
@@ -104,6 +104,7 @@ const LineSplitWithVoltageLevelDialog = ({
 
     const [newVoltageLevel, setNewVoltageLevel] = useState(null);
     const newVoltageLevelRef = useRef();
+    newVoltageLevelRef.current = newVoltageLevel;
```

Comment:
Could use 
`const newVoltageLevelRef = useRef(newVoltageLevel);`
instead of 
```
const newVoltageLevelRef = useRef();
newVoltageLevelRef.current = newVoltageLevel;
```

Reply:
. const newVoltageLevelRef = useRef(newVoltageLevel) : sets only the initial ref value 

. const newVoltageLevelRef = useRef(); 
  newVoltageLevelRef.current = newVoltageLevel;
  
  update the ref value at each render

Reply:
Done